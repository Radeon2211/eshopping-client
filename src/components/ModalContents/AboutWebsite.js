import React from 'react';
import Heading from '../UI/Heading/Heading';
import PlainText from '../UI/PlainText';

const AboutWebsite = () => {
  return (
    <>
      <Heading variant="h3" align="center" mgBottom="3">
        About website
      </Heading>
      <PlainText size="3" lineHeight="5" textAlign="justify">
        This website is not used to buy and sell products. Offers on this website are not real and
        orders will not be realized. Data that you specify in forms do not need to be real. However
        they are not processed in any way except of storing in database and displaying them on this
        website.
      </PlainText>
    </>
  );
};

export default AboutWebsite;
