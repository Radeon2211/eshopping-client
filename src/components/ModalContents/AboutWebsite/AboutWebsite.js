import Heading from '../../UI/Heading/Heading';
import PlainText from '../../UI/PlainText';

export default function AboutWebsite() {
  return (
    <>
      <Heading $variant="h3" $align="center" $mgBottom="level3">
        About website
      </Heading>
      <PlainText $size="level3" $lineHeight="level5" $textAlign="justify">
        This website is not used to buy and sell products. Offers on this website are not real and
        orders will not be realized. Data that you specify in forms do not need to be real. However
        they are not processed in any way except of storing in database and displaying them on this
        website.
      </PlainText>
    </>
  );
}
