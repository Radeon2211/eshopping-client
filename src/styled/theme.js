const theme = {
  colors: {
    light1: '#FAFAFA',
    light1Transparent: '#FAFAFA88',
    light2: '#DADFE1',
    light3: '#BFBFBF',
    light4: '#666',
    blue: '#1E8BC3',
    blueLight: '#22A7F0',
    blueDark: '#67809F',
    green: '#00A790',
    greenDark: '#009680',
    red: '#F22613',
  },
  shadows: {
    level1: '0 0 2px rgba(0, 0, 0, .1)',
    level2: '0 0 2px rgba(0, 0, 0, .3)',
    level3: '0 0 3px rgba(0, 0, 0, .3)',
    level4: '0 0 5px rgba(0, 0, 0, .3)',
    level5: '0 0 8px rgba(0, 0, 0, .3)',
  },
  fonts: {
    logo: 'Lobster, cursive',
    heading: 'Ubuntu, sans-serif',
    text: 'Lato, sans-serif',
  },
  spacings: {
    level1: '0.8rem',
    level2: '1.2rem',
    level3: '2.4rem',
    level4: '3.6rem',
    level5: '4.8rem',
  },
  zIndexes: {
    level1: 10,
    level2: 100,
    level3: 1000,
    level4: 10000,
  },
};

export default theme;
