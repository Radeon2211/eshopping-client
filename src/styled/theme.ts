const theme = {
  colors: {
    light1: '#FAFAFA',
    light1Transparent: '#FAFAFA88',
    light2: '#DADFE1',
    light3: '#BFBFBF',
    light4: '#777',
    blue: '#1E8BC3',
    blueLight: '#22A7F0',
    blueDark: '#67809F',
    green: '#06AD96',
    greenDark: '#009680',
    red: '#F22613',
    redLight: '#F94D2A',
    redDark: '#985A5A',
  },
  shadows: {
    level1: '0 0 3px rgba(0, 0, 0, 0.03)',
    level2: '0 0 3px rgba(0, 0, 0, 0.3)',
    level3: '0 0 5px rgba(0, 0, 0, 0.3)',
  },
  fonts: {
    logo: 'Lobster, cursive',
    heading: 'Noto Sans, sans-serif',
    text: 'Nunito, sans-serif',
  },
  spacings: {
    level1: '0.8rem',
    level2: '1.2rem',
    level3: '2.4rem',
    level4: '3.6rem',
    level5: '4.8rem',
  },
  zIndexes: {
    level1: 10,
    level2: 100,
    level3: 1000,
    level4: 10000,
    level5: 100000,
  },
  fontSizes: {
    level1: '1.3rem',
    level2: '1.4rem',
    level3: '1.5rem',
    level4: '1.7rem',
    level5: '2.2rem',
    level6: '2.6rem',
  },
  durations: {
    level1: 0.1,
    level2: 0.2,
    level3: 0.3,
  },
  lineHeights: {
    level1: '1',
    level2: '1.1',
    level3: '1.2',
    level4: '1.3',
    level5: '1.4',
    level6: '1.5',
  },
};

export type ThemeSpacing = keyof typeof theme.spacings;
export type ThemeZIndex = keyof typeof theme.zIndexes;
export type ThemeFontSize = keyof typeof theme.fontSizes;
export type ThemeDuration = keyof typeof theme.durations;
export type ThemeLineHeight = keyof typeof theme.lineHeights;

export default theme;
