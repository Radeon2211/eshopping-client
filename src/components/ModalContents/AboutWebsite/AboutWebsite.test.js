import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { ThemeProvider } from 'styled-components';
import AboutWebsite from './AboutWebsite';
import theme from '../../../styled/theme';

const setUp = () => {
  return render(
    <ThemeProvider theme={theme}>
      <AboutWebsite />
    </ThemeProvider>,
  );
};

afterEach(cleanup);

describe('<AboutWebsite />', () => {
  it('should render everything correctly', () => {
    const { asFragment } = setUp();
    expect(asFragment()).toMatchSnapshot();
  });
});
