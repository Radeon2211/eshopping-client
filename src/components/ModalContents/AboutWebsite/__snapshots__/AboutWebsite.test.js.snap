// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutWebsite /> should render everything correctly 1`] = `
<DocumentFragment>
  <h3
    class="sc-bdfBwQ iLqPaG"
  >
    About website
  </h3>
  <span
    class="sc-gsTCUz dplKcM"
    color=""
    display="inline-block"
    overflow="visible"
    size="3"
    spacing="0px"
    transform=""
  >
    This website is not used to buy and sell products. Offers on this website are not real and orders will not be realized. Data that you specify in forms do not need to be real. However they are not processed in any way except of storing in database and displaying them on this website.
  </span>
</DocumentFragment>
`;
